Assets/_Scripts/Projectile/Projectile.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(Rigidbody2D))]
+public class Projectile : MonoBehaviour
```

Comment:
I noticed that the Enemy abstract class PR also contains a separate `EnemyProjectile` [here](https://github.com/SolomanDavis/VGP145-S24-capstone-retro/pull/24/files#diff-0c7a0f53c33fe53b2d7092c22fa921f95b6f4154191e0391e847bf4bf83d05a9).

I believe they both act the same as in they both move at the exact same speed and only vertically. There's only 2 differences:
- PlayerProjectile moves upwards while EnemyProjectile moves downwards.
- PlayerProjectile kills enemies while EnemyProjectile kills player.

Once we merge this in it might be worth looking into consolidating them into one projectile script and updating the logic appropriately. Worth considering when we get to testing and want to update values of both these projectiles and ensure they perform the same.

(can leave this for now but something to come back to)